ion-schema/src/types.rs

```diff
@@ -780,50 +778,22 @@ mod type_definition_tests {
         /* For a schema with scale constraint as below:
             { scale: 2 }
         */
-        anonymous_type([scale(Int::I64(2).into())]),
-    TypeDefinitionKind::anonymous([Constraint::scale(Int::I64(2).into()), Constraint::type_constraint(34)])
+        anonymous_type([scale(2.into())]),
+    TypeDefinitionKind::anonymous([Constraint::scale(2.into()), Constraint::type_constraint(34)])
     ),
     case::exponent_constraint(
         /* For a schema with exponent constraint as below:
             { exponent: 2 }
         */
-        isl_type::v_2_0::anonymous_type([isl_constraint::v_2_0::exponent(Int::I64(2).into())]),
-    TypeDefinitionKind::anonymous([Constraint::exponent(Int::I64(2).into()), Constraint::type_constraint(34)])
+        isl_type::v_2_0::anonymous_type([isl_constraint::v_2_0::exponent(2.into())]),
+    TypeDefinitionKind::anonymous([Constraint::exponent(2.into()), Constraint::type_constraint(34)])
     ),
     case::timestamp_precision_constraint(
         /* For a schema with timestamp_precision constraint as below:
             { timestamp_precision: month }
         */
-        anonymous_type([timestamp_precision("month".try_into().unwrap())]),
-    TypeDefinitionKind::anonymous([Constraint::timestamp_precision("month".try_into().unwrap()), Constraint::type_constraint(34)])
-    ),
-    case::valid_values_constraint(
-        /* For a schema with valid_values constraint as below:
-            { valid_values: [2, 3.5, 5e7, "hello", hi] }
-        */
-        anonymous_type([valid_values_with_values(vec![2.into(), Decimal::new(35, -1).into(), 5e7.into(), "hello".to_owned().into(), Symbol::from("hi").into()]).unwrap()]),
-    TypeDefinitionKind::anonymous([Constraint::valid_values_with_values(vec![2.into(), Decimal::new(35, -1).into(), 5e7.into(), "hello".to_owned().into(), Symbol::from("hi").into()], IslVersion::V1_0).unwrap(), Constraint::type_constraint(34)])
-    ),
-    case::valid_values_with_range_constraint(
-        /* For a schema with valid_values constraint as below:
-            { valid_values: range::[1, 5.5] }
-        */
-        anonymous_type(
-            [valid_values_with_range(
```

Comment:
🗺️ The `valid_values_with_range` function no longer exists, so these test cases are no longer necessary.

Reply:
I think the test cases would still be necessary if we have different way to construct `valid_values` constraint with range, we should verify it here.